Display events

angel23v opened the pull request:
Display all the events once we click on "Iniciar" button

For more info see: https://github.com/BrightCoders-Institute/s5a1-administrador-de-eventos-team4/issues/11

angel23v committed:
Update Display Events

Updated the form as we display the events

Co-authored-by: Bryanroal <bryanroal27@gmail.com>

angel23v committed:
New Display Events

New view when we display the events

angel23v commented:
I pushed new changes

MauricioNC commented on app/views/events/index.html.erb:
```diff
@@ -0,0 +1,29 @@
+  <% end %>
+</div>
+</body>
+</html>
```
Add a new empty line after your code

MauricioNC commented on app/views/events/_navbar.html.erb:
```diff
@@ -0,0 +1,19 @@
+      </form>
+    </div>
+  </div>
+</nav>
```
Same, add a new empty line after your code

MauricioNC commented on app/controllers/events_controller.rb:
```diff
@@ -0,0 +1,28 @@
+    @event = Event.new(events_paramas)
+
+    if @event.save
+      redirect_to '/views/events/index'
```
You can use the prefixes that rails gives you with resources.

See  the documentation at:
https://guides.rubyonrails.org/routing.html#resource-routing-the-rails-default

MauricioNC commented on app/controllers/events_controller.rb:
```diff
@@ -0,0 +1,28 @@
+  private
+
+  def events_paramas
+    params.require(:event).permit(:Titulo, :Descripcion, :Fecha, :Ubicacion, :Costo)
```
The fields in a table must be lowercase.
You can run a migration to rename this fields.

See the documentation at:
https://edgeguides.rubyonrails.org/active_record_migrations.html
https://edgeapi.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-rename_column

MauricioNC replied:
The fields in a table must be lowercase
You can run a migration to rename this fields.

See the documentation at:
https://edgeguides.rubyonrails.org/active_record_migrations.html
https://edgeapi.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-rename_column

angel23v committed:
Rename Attributes

Adding a new empty line after code, rename the attributes in lowercase, change action when an event is save

MauricioNC commented on app/views/events/main.html.erb:
```diff
@@ -1,33 +1,16 @@
+</div>
 </body>
-</html>
+</html>
```
Add new empty line at the end of your code

angel23v committed:
Empty line

Adding missing empty lines

Co-authored-by: Bryanroal <bryanroal27@gmail.com>

juperameza reviewed (approved):
LGTM

angel23v committed:
Add files
